src/string.ts

```diff
@@ -116,3 +116,17 @@ export const template = (
     return acc.replace(match[0], data[match[1]])
   }, str)
 }
+
+/**
+ * Replace international characters with their normalized equivalent by removing diacritics.
+ *
+ * Ex. removeDiacritics('áéíóú') -> 'aeiou'
+ * Ex. removeDiacritics('àèìòù') -> 'aeiou'
+ * Ex. removeDiacritics('âêîôû') -> 'aeiou'
+ * Ex. removeDiacritics('ãẽĩõũ') -> 'aeiou'
+ * Ex. removeDiacritics('äëïöü') -> 'aeiou'
+ */
+export const removeDiacritics = (str: string | null | undefined): string => {
```

Comment:
I don't think this name will mean much to many people... With the name `removeDiacritics` it seems like a function only people explicitly working with international languages would use. I do think there is some global utility here for anyone who needs to clean a string to be used in a path/folder/file. 

So, what if we just call it `clean`? 

I'm open to your thoughts!

Reply:
I think 'clean' is too vague.
My proposal covers a specific case while clean can mean different things : clean path chars, clean special chars, clean dangerous html tags, clean xxxx

English is not mother tongue, so I'm not sure of my wording, but maybe 'cleanAccented' is more widely understand able still keeping the purpose clear ?

Reply:
I've pushed the changes, if you agree with the new proposed name

Reply:
Hello @rayepps 

I tried to upgrade the function in the way you suggested (but usign enum flags).

let me know if it fits in the library convention

Reply:
Hey @stevebeauge sorry for the delayed response, my day job has been a bit nuts lately. Enums are a no go 😬 the pro/con between enums and union types is a clear win for union types: they produce the same type safety with no additional code.

I realize this is my opinion, and I know its one that can be argued (as some of my peers have 😅 -- talking to you @zkirby ), but since radash currently has 0 enums I think it's best to keep it that way.